maintenance/Maintenance.php

```diff
@@ -1113,17 +1113,17 @@ public function setDB( &$db ) {
 
 	/**
 	 * Lock the search index
-	 * @param &$db Database object
+	 * @param &$db DatabaseBase object
 	 */
 	private function lockSearchindex( &$db ) {
-		$write = array( 'searchindex' );
+		$write = []; # array( 'searchindex' ); - Wikia change, there's no searchindex per-wiki table anymore
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)